src/game_renderer/GameRenderer.h

```diff
@@ -3,16 +3,39 @@
 #ifndef GAME_RENDERER_H_
 #define GAME_RENDERER_H_
 
+#include <cmath>
+#include <fstream>
+#include <glm/gtc/type_ptr.hpp>
+#include <glm/gtc/matrix_transform.hpp>
+#include <glob.h>
+#include <iostream>
+#include <unordered_map>
 #include "game_state/GameState.h"
+#include "GLFW/glfw3.h"
+#include "GL/glew.h"
+#include "GLSL.h"
+#include "json.hpp"
+#include "MatrixStack.h"
+#include "Program.h"
+#include "Texture.h"
```

Comment:
A lot of headers in chromium will actually forward-declare symbols just to get around including more headers from a header like this: https://cs.chromium.org/chromium/src/content/public/renderer/render_frame_observer.h?q=renderframeobserver&sq=package:chromium&l=22